INSTALL.md

```diff
@@ -101,7 +101,7 @@ sudo service postgresql start
 ```
 ### PostgreSQL installation on Mac OS X
 
-For Mac OS X, you can either install postgres through the [Postgres.app](https://postgresapp.com/downloads.html). These instructions were tested with the Postgres.app v9.5, v13.
+For Mac OS X, you can install postgres through the [Postgres.app](https://postgresapp.com/downloads.html). These instructions were tested with the Postgres.app v9.5 and v13.
```

Comment:
I would go a little further and reword it to be, "For Mac OS X it is strongly encouraged to install Postgres through the Postgres.app."

IMOO, the second line implies that there are separate versions of the Postgres.app, namely 9.5 and 13. This is not the case. The Postgres.app allows you to dynamically create and run one or more instances of Postgres, each of which can be a different version. We should state instead that Qiita is supported on versions 9.5 and 13 of PostgreSQL itself, not the Postgres.app.